lib/base_service.ts

```diff
@@ -51,11 +53,20 @@ export interface Credentials {
   username: string;
   password: string;
   api_key: string;
+  platform_api_key?: string;
```

Comment:
I suspect you're already working on this change, but I think we settled on something like `iam_apikey` instead.